src/main/java/com/adyen/service/Resource.java

```diff
@@ -51,6 +51,18 @@ public Resource(Service service, String endpoint, List<String> requiredFields) {
         this.requiredFields = requiredFields;
     }
 
+    /**
```

Comment:
We can just put it directly under the resource file I guess?

Reply:
Indeed, ideally this file should be under `package com.adyen.service.resource`. If we move it, other 95 classes need to be updated (via IDE refactor). This is a `public` class, maybe merchants use it for something? It would be a breaking change (i.e. for v19).